2021-03-23 17:12:12.210	1	INFO	log/demo_log_2.go:5	111111111111
2021-03-23 17:12:35.195	1	INFO	log/demo_log_2.go:5	111111111111
2021-03-23 17:12:35.208	1	INFO	log/demo_log.go:56	
----------
Traceback (most recent call last):
	[Tance] 0x10000|InternalError
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 42 ----> jpkt/src/core.(*JpktStandardError).createErrCtx
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 131 ----> jpkt/src/core.RaiseStandardError
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 139 ----> jpkt/src/core.StandardizeErr
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 55 ----> main.test4
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 79 ----> main.main
	File "<D:/env/go/1.14.4/src/runtime/proc.go>", Line 203 ----> runtime.main
	File "<D:/env/go/1.14.4/src/runtime/asm_amd64.s>", Line 1373 ----> runtime.goexit
	......
####Code: "0x10000", Type: "InternalError"
####Debug: "EEEEEEEEEEEEEEE", Desc: "内部错误，错误代码：0x10000"
----------

2021-03-23 17:12:35.208	7	INFO	log/demo_log.go:56	
----------
Traceback (most recent call last):
	[Tance] 0x10000|InternalError
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 42 ----> jpkt/src/core.(*JpktStandardError).createErrCtx
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 131 ----> jpkt/src/core.RaiseStandardError
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 139 ----> jpkt/src/core.StandardizeErr
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 55 ----> main.test4
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 35 ----> main.test3
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 31 ----> main.test2
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 27 ----> main.test1
	File "<D:/env/go/1.14.4/src/runtime/asm_amd64.s>", Line 1373 ----> runtime.goexit
	......
####Code: "0x10000", Type: "InternalError"
####Debug: "EEEEEEEEEEEEEEE", Desc: "内部错误，错误代码：0x10000"
----------

2021-03-23 17:12:35.208	1	WARN	core/exception.go:181	
----------
Traceback (most recent call last):
	[Tance] 0x10001|UnknownError
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 42 ----> jpkt/src/core.(*JpktStandardError).createErrCtx
	File "<D:/workspace/jrsa/Jpkt/src/core/exception.go>", Line 131 ----> jpkt/src/core.RaiseStandardError
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 61 ----> main.test5
	File "<D:/workspace/jrsa/Jpkt/demo/log/demo_log.go>", Line 80 ----> main.main
	File "<D:/env/go/1.14.4/src/runtime/proc.go>", Line 203 ----> runtime.main
	File "<D:/env/go/1.14.4/src/runtime/asm_amd64.s>", Line 1373 ----> runtime.goexit
	......
####Code: "0x10001", Type: "UnknownError"
####Debug: "Can't do this thing...", Desc: "内部异常，代码[0x10001]"
----------

